2_process/src/process_functions.R

```diff
@@ -0,0 +1,53 @@
+# Mackenzie Marti (mmarti-usgs, mmarti@usgs.gov)
+#
+# This script defines functions related to processing data and saving processed
+# data.
+
+library(dplyr)
+library(readr)
+library(stringr)
+library(sbtools)
+library(whisker)
+
+# prepare/clean data for plotting
+prep_data <- function(data, project_output_dir) {
```

Comment:
Similarly, it is better to use the direct output path to file for the input and not out put file and specifying file name later.

Reply:
Updated.